### PR TITLE
Update team page for Team HT

### PR DESCRIPTION
**Pull request by xinan**
:smile: Ready to go!

### CODE REVIEW

**Comment by lowjoel** [comment]
@xinan can you please help remove the dummy filler text in every section?

**Comment by xinan** [comment]
Oh okay done! :smile: 

Actually I left them there intentionally so that I could refer to them to know what to fill up there. But yeah, I could remove them and refer to the template instead when needed.

**Comment by lowjoel** [comment]
@xinan sorry, some confusion about the templates came up. Can you monitor #3 and see if @damithc is okay with the new page layout?

Sorry for the extra work >.<

I'll post here again after Damith confirms that the new template is easier to understand.

**Comment by xinan** [comment]
Oh sure. No problem!

**Comment by lowjoel** [comment]
@xinan sorry, we've just updated the templates. Can you sync up with the newest templates please? 

When in doubt, see the existing pages for SocketIO and CodeBender. Thanks!

File: teams/hubturbo.md [hunk 1]
```diff
@@ -2,4 +2,84 @@
...
+* [x] First PR merged - Week 1, Thu
+
+##### Contribution to project HT
+
+##### Contribution to project Y
+
+##### Technology expertise in tool Z
+
+
+# Technology Expertise
```
Comment (lowjoel) [inline teams/hubturbo.md]: Tech expertise is now together with the team summary at the top

**Comment by xinan** [comment]
@lowjoel :smile: Updated!

**Comment by lowjoel** [comment]
Sweet.

**Pull request merged**